src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -748,6 +748,16 @@ private function resolveServices($value, $file, $isParameter = false)
 
                 return new Reference($id);
             }
+            if ('yaml_file' === $value->getTag()) {
+                $rootDir = \dirname($file);
+                $file = \is_file($argument) ? $argument : "$rootDir/$argument";
```

Comment:
this won't support using `%kernel.project_dir%/some_file.yaml`, as it won't be resolved yet here.

Reply:
It was the case before but @nicolas-grekas suggested that we should not resolve parameters (see above)

Reply:
he suggested not to resolve **env placeholders**. Resolving parameters is not the same.

Reply:
Fixed!